tests/Message/SIMCompleteAuthorizeRequestTest.php

```diff
@@ -45,7 +45,7 @@ public function testSend()
         $this->request->setApiLoginId('user');
         $this->request->setHashSecret('shhh');
         $this->request->setAmount('10.00');
-        $this->request->setTransactionId(99);
+        $this->request->setTransactionReference(99);
```

Comment:
You wouldn't ever *set* the transactionReference here; you would just *get* it for storage. It would come from the x_trans_id POSTed to this notify handler.